sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Customizations/Models/ConversionErrorCode.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion.Models
+{
+    /// <summary> The ConversionErrorCode. </summary>
+    public readonly partial struct ConversionErrorCode
+    {
+        /// <summary> UNKNOWN. </summary>
+        [CodeGenMember("Unknown")]
+        public static ConversionErrorCode Unknown { get; } = new ConversionErrorCode(UnknownValue);
+        /// <summary> NO_ERROR. </summary>
+        [CodeGenMember("NOError")]
+        public static ConversionErrorCode NoError { get; } = new ConversionErrorCode(NoErrorValue);
+        /// <summary> SERVICE_ERROR. </summary>
+        [CodeGenMember("ServiceError")]
+        public static ConversionErrorCode ServiceError { get; } = new ConversionErrorCode(ServiceErrorValue);
+        /// <summary> INVALID_ASSET_URI. </summary>
+        [CodeGenMember("InvalidAssetURI")]
+        public static ConversionErrorCode InvalidAssetUri { get; } = new ConversionErrorCode(InvalidAssetUriValue);
+        /// <summary> INVALID_JOB_ID. </summary>
+        [CodeGenMember("InvalidJOBID")]
+        public static ConversionErrorCode InvalidJobId { get; } = new ConversionErrorCode(InvalidJobIdValue);
+        /// <summary> INVALID_GRAVITY. </summary>
+        [CodeGenMember("InvalidGravity")]
+        public static ConversionErrorCode InvalidGravity { get; } = new ConversionErrorCode(InvalidGravityValue);
+        /// <summary> INVALID_SCALE. </summary>
+        [CodeGenMember("InvalidScale")]
+        public static ConversionErrorCode InvalidScale { get; } = new ConversionErrorCode(InvalidScaleValue);
+        /// <summary> ASSET_SIZE_TOO_LARGE. </summary>
+        [CodeGenMember("AssetSizeTOOLarge")]
```

Comment:
Similar question about the casing.

Reply:
So autogen for some reason turned ASSET_SIZE_TOO_LARGEinto AssetSizeTOOLarge when generating the extensible enum, I guess it thought anything with <4 letters in that uppercase underscore-delimited error code string was an acronym? The custom partial class was for renaming them with proper casing

Reply:
/cc @ShivangiReja @pakrym is this a known issue?

Reply:
cc @chamons

Reply:
I know we have special logic in namer to process acronyms. @timotheeguerin does this sound familiar?

Reply:
yeah modelerfour naming, will keep upper words together up to a certain length. there is way to configure around if this is not what you want. In this case I think Just chancing the x-ms-client-name to be `AssetSizeTooLarge` or `Asset_Size_Too_Large` should have it done correctly.

Reply:
@jastenze can we use swagger transforms to set the x-ms-client-name instead of the current approach?

Reply:
Fixed through swagger change